src/hooks.server.ts

```diff
@@ -1,8 +1,14 @@
+import * as Sentry from '@sentry/sveltekit';
 import type { Handle } from '@sveltejs/kit';
 import redirects from './redirects.json';
 import { sequence } from '@sveltejs/kit/hooks';
 import { BANNER_KEY } from '$lib/constants';
 
+Sentry.init({
+    dsn: "https://27d41dc8bb67b596f137924ab8599e59@o1063647.ingest.us.sentry.io/4507497727000576",
```

Comment:
Same here

Reply:
If we dont do it as configurable variable from ENV (GitHub secrets), we should make it const and re-use, so we can update it in 1 place in future